src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -117,13 +120,15 @@ private DownloadMapsWindow(
       progressPanel.download(pendingDownloads);
     }
 
+    pendingDownloads.addAll(ClientContext.DOWNLOAD_COORDINATOR.getDownloads().stream()
```

Comment:
Nice.  I was wondering what `DownloadCoordinator#getDownloads()` could be used for. :smile:  One question, though...  It looks like it returns "terminated" downloads, as well.  That's currently always going to be empty, but could it cause a problem in the future when we implement download cancellation?